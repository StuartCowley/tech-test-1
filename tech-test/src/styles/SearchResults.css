.search-results {
  display: flex;
  flex-direction: row;
  flex-wrap: wrap;
  justify-content: space-evenly;
}

.card-image {
  height: 100px;
  width: 100px;
  padding: 5px;
}

.card-image:hover {
  transform: scale(1.2) ;
}