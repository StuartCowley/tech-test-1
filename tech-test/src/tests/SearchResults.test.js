import React from "react";
import { render } from "@testing-library/react";
import SearchResults from "../components/SearchResults";

describe("SearchResults", () => {
  const validProps = {
    results: ["img1", "img2", "img3"]
  };
  
  it("renders SearchResult component correctly", () => {
    const { asFragment } = render(
    <SearchResults
      results={validProps.results}
    />)

    expect(asFragment()).toMatchSnapshot();
  });
});