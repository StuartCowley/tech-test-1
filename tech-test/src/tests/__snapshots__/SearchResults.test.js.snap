// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchResults renders SearchResult component correctly 1`] = `
<DocumentFragment>
  <div
    class="search-results"
  >
    <img
      alt="result-tile"
      class="card-image"
      src="img1"
    />
    <img
      alt="result-tile"
      class="card-image"
      src="img2"
    />
    <img
      alt="result-tile"
      class="card-image"
      src="img3"
    />
  </div>
</DocumentFragment>
`;
